### PR TITLE
Support for sai header 1.10.2 and sai 7.0.0.6

### PR DESCRIPTION
**Pull request by vmittal-msft**
Signed-off-by: Vineet Mittal <vineetmittal@microsoft.com>

### CODE REVIEW

File: make/Make.config [hunk 1]
```diff
@@ -120,12 +120,7 @@ ifndef SDKBUILD
 SDKBUILD :=build
 endif
 
-ifdef SDK_OUTDIR
```
Comment (tjchadaga) [inline make/Make.config]: these changes were added by brcm in 6.1.0.3.. was this removed in 7.0.0.6 or is this our change?

Comment (vmittal-msft) [reply make/Make.config]: Good catch. Fixed it. Thanks.

**Comment by judyjoseph** [comment]
@vmittal-msft In this PR https://github.com/Azure/sonic-buildimage/pull/10583, I see more changes to sai-bcm-modules. Shouldn't those changes be here too, please confirm,

**Comment by vmittal-msft** [comment]
Compared this with BRCM drop again and there are no changes on DNX side, only XGS. So we are ok.

**Review by judyjoseph — APPROVED** [review]

**Review by tjchadaga — APPROVED** [review]

**Pull request merged**